index.html

```diff
@@ -4099,34 +4099,36 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
 
         </section>
 
-        <section id="mLUm-chunk">
-          <h2><span class="chunk">mLUm</span> Metadata for Maximum Single-Pixel and Frame-Average Luminance Levels</h2>
+        <section id="cLLi-chunk">
+          <h2><span class="chunk">cLLi</span> Content Light Level Information</h2>
           <p>The four-byte chunk type field contains the hexadecimal values</p>
 
           <pre>
 <!-- 109 76 85 109 -->6D 4C 55 6D
 </pre>
         
-          <p>If present, the <span class="chunk">mLUm</span> chunk identifies 
-             the brightest single pixel or "Maximum Content Light Level" (MaxCLL) for the entire 
-             length of content (in every frame) and is typically clipped based on the mastering displays 
-             peak luminance specification. (MaxCLL) is often 
-             algorithmically analyzed to avoid false values generated by processing or noise from 
-             a single pixel values luminance level. The "Maximum Frame Average Light Level" (MaxFALL) value 
-             is determined by measuring each frame's average light level. The frame 
-             with the highest average light level becomes the MaxFALL value.  
-             For static PNG, 
-             the <a>static image</a> is analyzed 
-             and is considered to be the first (and only) frame. 
-             For animated PNG, 
-             each frame of the 
-             <a href="#apng-frame-based-animation">frame-based animation</a> sequence is analyzed.
-          </p>
-
-           <p>The following specifies the syntax of the <span class="chunk">mLUm</span> chunk:</p>
-
-           <table id="mLUm-chunk-syntax" class="numbered simple">
-             <caption>mLUm chunk components</caption>
+          <p>If present, the <span class="chunk">cLLi</span> chunk identifies two characteristics:</p>
+             
+          <p>MaxFALL (Maximum Frame Average Light Level) indicates the maximum value of the frame 
+             average light level (in cd/m2 or nits) of the entire playback sequence. MaxFALL is 
+             calculated by averaging the decoded luminance values of all the pixels within a frame.</p>
```

Comment:
"within *a* frame" sounds like it is measuring a single frame.

Can we reword this? Something like "All the pixels of a given frame have their luminance values averaged. Then the maximum of those frame averages is stored." (I'm not wording this well here.)

I'm worried that "...averaging the decoded luminance values of all the pixels of all frames" sounds like one big average, instead of one max of many averages.